---
pr2_motor_diagnostic_controller/MotorSample[] sample_buffer
